VetClinicApi.Application/Services/CustomerHandlig/CustomerService.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using VetClinicApi.Application.Validation;
+using VetClinicApi.Core.Entities;
+using VetClinicApi.Database.Repositories;
+
+namespace VetClinicApi.Application.Services.CustomerHandlig
+{
+    public class CustomerService : ICustomerService
+    {
+        private readonly IAbstractRepository<Customer> _repository;
+        private CustomerValidator _fieldsValidator = new CustomerValidator();
```

Comment:
Extract generic interface and provide this interface as parameter of constructior

Reply:
Also, add pair interface-class into ApplicationDependencyInjection.cs as scoped